.gitignore

```diff
@@ -22,8 +22,7 @@ yarn-error.log*
 *.sw?
 
 # Custom config files.
-src/config/endpoints.json
-src/config/peers-api.json
+src/config/setup.json
```

Comment:
doesnt it use the peers-api.json file anymore? If not, we need to know how it changed because it will affect the build process (before automatic build, we copy the peers-api.json into the repo).

Reply:
peers-api.json and endpoints.json already merged into `setup.json`.

it wont affect the build  process, because all already predefine in `setup.json`.

but if you want make some change, you can edit `setup.json`

view more on template file [setup.json.mt](https://github.com/AnthonyLaw/nf-catapult-explorer/blob/rebranding/src/config/setup.json.mt)

Reply:
it will affect the jenkins build process ; not the package build